web/src/app.d.ts

```diff
@@ -9,3 +9,11 @@ declare namespace App {
 
 	// interface Platform {}
 }
+
+// Source: https://stackoverflow.com/questions/63814432/typescript-typing-of-non-standard-window-event-in-svelte
+// To fix the <svelte:window... in components/asset-viewer/photo-viewer.svelte
+declare namespace svelte.JSX {
+	interface HTMLAttributes<T> {
+		oncopyImage?: () => void;
+	}
+}
```

Comment:
Is this the right way to fix that type issue? Feels like I'm doing some global thing here, but maybe that's just how it is.

Reply:
Turns out this fixes the typescript error but causes an eslint one:
```
/home/runner/work/immich/immich/web/src/app.d.ts
  16:12  warning  'HTMLAttributes' is defined but never used  @typescript-eslint/no-unused-vars
  16:27  warning  'T' is defined but never used               @typescript-eslint/no-unused-vars
```